src/test/regress/expected/role_operations_from_non_maindb.out

```diff
@@ -0,0 +1,67 @@
+-- Create a new database
+set citus.enable_create_database_propagation to on;
+CREATE DATABASE test_db;
+SET citus.superuser TO 'postgres';
+-- Connect to the new database
+\c test_db
+-- Test CREATE ROLE with various options
+CREATE ROLE test_role1 WITH LOGIN PASSWORD 'password1';
+\c test_db - - :worker_1_port
+CREATE USER "test_role2-needs\!escape"
+WITH
+    SUPERUSER CREATEDB CREATEROLE INHERIT LOGIN REPLICATION BYPASSRLS CONNECTION
+LIMIT 10 VALID UNTIL '2023-01-01' IN ROLE test_role1;
+\c regression - - :master_port
+select result FROM run_command_on_all_nodes($$
+    SELECT array_to_json(array_agg(row_to_json(t)))
+    FROM (
+        SELECT rolname, rolsuper, rolinherit, rolcreaterole, rolcreatedb,
+        rolcanlogin, rolreplication, rolbypassrls, rolconnlimit,
+        (rolpassword != '') as pass_not_empty, DATE(rolvaliduntil)
+        FROM pg_authid
+        WHERE rolname in ('test_role1', 'test_role2-needs\!escape')
+        ORDER BY rolname
+    ) t
+$$);
```

Comment:
After creating those two roles, let's show that pg_dist_object has a record for those roles by using run_command_on_all_nodes.

Reply:
Added